react-frontend/src/listings/EditListing.js

```diff
@@ -80,6 +81,13 @@ function EditListing() {
       ...prevListing,
       [name]: value
     }));
+    if(name === "title" && value === ""){
+      setErrorMessage('Title is required.');
+    } else if(name === "price" && value === ""){
+      setErrorMessage('Price is required.');
+    } else{
```

Comment:
Seems like no need to have the last else block, the errorMessage is already set to that empty string when you're initially declaring it. For reference, any useState() will set the initial state to whatever is put in parantheses.